docs/user_documentation/guide-upload.md

```diff
@@ -131,6 +131,8 @@ String url
 entityTypeId: The name of the entity, this only has effect for VCF files (.vcf and .vcf.gz)
 	Default value is the filename.
 
+packageId: The id of the package you want the imported file to be placed under. If package does not exist, an error will be thrown. Not supplying the package will place an imported file under the default package 'base'
```

Comment:
Add some documentation here that this only has effect for VCF files (like the entityTypeId parameter).